### ISSUE TITLE
Security alert level labels have hard to read text color

### ISSUE THREAD

**Issue by xt0rted** [StylishThemes/GitHub-Dark#758]
These alerts have [4 levels](https://help.github.com/articles/about-security-alerts-for-vulnerable-dependencies/#githubs-security-alerts-for-vulnerable-dependencies):
- Low
- Moderate
- High
- Critical

None of my repos have a critical alert so I'm not sure what classes that one is using. I'm guessing they're using `Label bg-red` which if so looks fine. I've asked someone at GitHub what this one uses but haven't heard back (yet).

* **Browser**: chrome
* **Operating System**: win10
* **URL**: https://github.com/StylishThemes/GitHub-Dark/network/alerts (may or may not have content for this repo)
* **Screenshot**:

![image](https://user-images.githubusercontent.com/831974/47057603-7481eb00-d18f-11e8-9ccc-f87724a9aac0.png) ![image](https://user-images.githubusercontent.com/831974/47057464-e4dc3c80-d18e-11e8-9b7b-52cf422af24a.png)

![image](https://user-images.githubusercontent.com/831974/47057651-a4c98980-d18f-11e8-9b86-08bfdd00e9ae.png) ![image](https://user-images.githubusercontent.com/831974/47057144-72b72800-d18d-11e8-937f-ac89df645cf7.png)

![image](https://user-images.githubusercontent.com/831974/47057626-895e7e80-d18f-11e8-8306-fc323560ab5c.png) ![image](https://user-images.githubusercontent.com/831974/47057494-02a9a180-d18f-11e8-81c5-6c142bf897c7.png)

* **HTML of the section where the issue occurs**:

```html
<span class="Label bg-yellow-light text-gray-dark p-1 float-right">low severity</span>
```

````html
<span class="Label bg-yellow text-gray-dark p-1 float-right">moderate severity</span>
````

```html
<span class="Label Label--orange p-1 float-right">high severity</span>
```

**Comment by silverwind** [comment]
Can you also post the HTML of the whole alert box when it has content? We also need to fix the tranparent background there.

**Comment by xt0rted** [comment]
@silverwind here you go

![image](https://user-images.githubusercontent.com/831974/47113786-ea399580-d227-11e8-8b3f-e1db456c3d59.png)


![image](https://user-images.githubusercontent.com/831974/47113764-d8f08900-d227-11e8-9302-21c68702a36f.png)


```html
<div class="column three-fourths">
      
  <div id="dependencies">
    <div class="Subhead">
      <h2 class="Subhead-heading">Alerts</h2>
    </div>
  </div>

  <div class="table-list-header">
    <div class="table-list-filters">
      <div class="table-list-header-toggle states float-left pl-3">
        <a href="/MyOrg/MyProject/network/alerts?q=is%3Aopen" class="btn-link ">
          <svg class="octicon octicon-alert" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M8.893 1.5c-.183-.31-.52-.5-.887-.5s-.703.19-.886.5L.138 13.499a.98.98 0 0 0 0 1.001c.193.31.53.501.886.501h13.964c.367 0 .704-.19.877-.5a1.03 1.03 0 0 0 .01-1.002L8.893 1.5zm.133 11.497H6.987v-2.003h2.039v2.003zm0-3.004H6.987V5.987h2.039v4.006z"></path></svg>
          0 Open
        </a>

        <a href="/MyOrg/MyProject/network/alerts?q=is%3Aclosed" class="btn-link selected">
          <svg class="octicon octicon-check" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M12 5l-8 8-4-4 1.5-1.5L4 10l6.5-6.5L12 5z"></path></svg>
          1 Closed
        </a>
      </div>

      <div class="table-list-header-toggle float-right">
        <details class="details-reset details-overlay select-menu float-left">
          <summary class="btn-link select-menu-button" aria-haspopup="true">
            Sort<span class="sort-label"></span>
          </summary>

          <div class="select-menu-modal position-absolute right-0" role="menu" style="z-index: 1000;" aria-label="Sort by">
            <div class="select-menu-header">
              <span class="select-menu-title">Sort by</span>
            </div>
            <div class="select-menu-list">
                <a href="/MyOrg/MyProject/network/alerts?q=is%3Aclosed+sort%3Anewest" class="select-menu-item selected" aria-checked="true" role="menuitemradio">

                   <svg class="octicon octicon-check select-menu-item-icon" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M12 5l-8 8-4-4 1.5-1.5L4 10l6.5-6.5L12 5z"></path></svg>
                   <div class="select-menu-item-text">Newest</div>
                </a>
                <a href="/MyOrg/MyProject/network/alerts?q=is%3Aclosed+sort%3Aoldest" class="select-menu-item " role="menuitemradio">

                   <svg class="octicon octicon-check select-menu-item-icon" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M12 5l-8 8-4-4 1.5-1.5L4 10l6.5-6.5L12 5z"></path></svg>
                   <div class="select-menu-item-text">Oldest</div>
                </a>
                <a href="/MyOrg/MyProject/network/alerts?q=is%3Aclosed+sort%3Aseverity" class="select-menu-item " role="menuitemradio">

                   <svg class="octicon octicon-check select-menu-item-icon" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M12 5l-8 8-4-4 1.5-1.5L4 10l6.5-6.5L12 5z"></path></svg>
                   <div class="select-menu-item-text">Severity</div>
                </a>
                <a href="/MyOrg/MyProject/network/alerts?q=is%3Aclosed+sort%3Amanifest-path" class="select-menu-item " role="menuitemradio">

                   <svg class="octicon octicon-check select-menu-item-icon" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M12 5l-8 8-4-4 1.5-1.5L4 10l6.5-6.5L12 5z"></path></svg>
                   <div class="select-menu-item-text">Manifest path</div>
                </a>
                <a href="/MyOrg/MyProject/network/alerts?q=is%3Aclosed+sort%3Apackage-name" class="select-menu-item " role="menuitemradio">

                   <svg class="octicon octicon-check select-menu-item-icon" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M12 5l-8 8-4-4 1.5-1.5L4 10l6.5-6.5L12 5z"></path></svg>
                   <div class="select-menu-item-text">Package name</div>
                </a>
            </div>
          </div>
        </details>
      </div>
    </div>
  </div>

  <div class="border-right border-bottom border-left">
    <ul>
        <li class="Box-row Box-row--focus-gray p-0 selectable read">
          <div class="d-table table-fixed width-full Box-row--drag-hide position-relative">
            <div class="float-left pt-2 pl-3">
              <span class="tooltipped tooltipped-n" aria-label="Dismissed alert">
                <svg class="octicon octicon-mute closed" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M8 2.81v10.38c0 .67-.81 1-1.28.53L3 10H1c-.55 0-1-.45-1-1V7c0-.55.45-1 1-1h2l3.72-3.72C7.19 1.81 8 2.14 8 2.81zm7.53 3.22l-1.06-1.06-1.97 1.97-1.97-1.97-1.06 1.06L11.44 8 9.47 9.97l1.06 1.06 1.97-1.97 1.97 1.97 1.06-1.06L13.56 8l1.97-1.97z"></path></svg>
              </span>
            </div>

            <div class="float-left lh-condensed p-2">
              <a href="/MyOrg/MyProject/network/alert/src/MyProj/package-lock.json/lodash/closed" class="link-gray-dark v-align-middle no-underline h4">
                lodash
              </a>

              <div class="mt-1 text-small text-gray">
                <span class="opened-by">
                  opened
                  <relative-time datetime="2018-08-13T21:16:14Z" title="Aug 13, 2018, 5:16 PM EDT">on Aug 13</relative-time>
                  by
                  <a class="muted-link" role="author" href="https://github.com/github">GitHub</a>
                  •
                  <a class="muted-link" href="/MyOrg/MyProject/blob/master/src/MyProj/package-lock.json">src/MyProj/package-lock.json</a>
                </span>
              </div>
            </div>

            <div class="float-right px-3 py-2">
              <span class="Label Label--orange p-1 float-right">high severity</span>
            </div>
          </div>
        </li>
    </ul>
  </div>

  <div class="paginate-container">
    
  </div>


  <div class="protip">
    <p>GitHub tracks known security vulnerabilities in some dependency manifest files. <a href="https://help.github.com/articles/about-security-alerts-for-vulnerable-dependencies" target="_blank">Learn more about alerts.</a></p>
  </div>

</div>
```

**Comment by xt0rted** [comment]
@silverwind I just realized this was closed, but with the latest version the `moderate severity` label is harder to read I think (`low severity` looks better though).

**Low**

![image](https://user-images.githubusercontent.com/831974/47638879-40ff7300-db36-11e8-9508-56a0257af863.png)

**Moderate**

![image](https://user-images.githubusercontent.com/831974/47638869-37760b00-db36-11e8-8ddd-00d28a67360d.png)
